CODEOWNERS

```
@@ -1 +1 @@
-* @saqadri @rholinshead @suyoglastmileai @LastMileNewGrad @jonathanlastmileai @rossdanlm
\ No newline at end of file
+* @saqadri @rholinshead @suyoglastmileai @LastMileNewGrad @jonathanlastmileai @rossdanlmrite
```

```suggestion
* @saqadri @rholinshead @suyoglastmileai @Ankush-lastmile @jonathanlastmileai @rossdanlm
```

Looks like Ankush updated his username and accidental `rite` suffix on Rossdan's